ratis-server/src/main/java/org/apache/ratis/server/impl/RoleInfo.java

```diff
@@ -44,8 +45,10 @@
   private final AtomicReference<FollowerState> followerState = new AtomicReference<>();
   /** Used when the peer is candidate, to request votes from other peers */
   private final AtomicReference<LeaderElection> leaderElection = new AtomicReference<>();
+  private final AtomicBoolean pauseLeaderElection = new AtomicBoolean(false);
 
   private final AtomicReference<Timestamp> transitionTime;
+  private final AtomicBoolean pauseElection = new AtomicBoolean(false);
```

Comment:
This should be removed.

Reply:
thanks for reviewing code,  I have remove the extra code